backend/translation/app/translate.py

```diff
@@ -28,7 +28,7 @@ def translate_to_lang(data: dict, from_language: str, to_language: str) -> dict:
     if from_language not in language_codes:
         translation =  "Not a viable language"
         return translation
-    elif from_language != "en":
+    elif from_language != "en" and to_language != "en":
```

Comment:
ni får gärna försöka förklara varför ni inte tycker den här ändringen borde matas in. Jag vet ju faktiskt inte vad som blir knas, om det har med en till en översättningen som gör det, isåfall behövs det nog en koll för det också.

Det är bara att frontenden tillåter inte översättning originalspråk till originalspråk just nnu, och mina tester i frontenden har visat på att det funkar